packages/controllers/src/snaps/SnapController.test.ts

```diff
@@ -342,7 +355,19 @@ describe('SnapController', () => {
   });
 
   it('should add a snap and use its JSON-RPC api with a WebWorkerExecutionService', async () => {
-    const [snapController] = getSnapControllerWithEES();
+    const messenger = getSnapControllerMessenger();
+    jest.spyOn(messenger, 'call').mockImplementation((method, ...args) => {
+      if (
+        method === 'PermissionController:hasPermission' &&
+        args[1] === LONG_RUNNING_PERMISSION
+      ) {
+        return false;
+      }
+      return true;
+    });
```

Comment:
This boilerplate is really blowing up this test file. Is there a neat way to encapsulate this within `getSnapControllerMessenger()`?

Reply:
I don't know how I feel about it, but I have added a default spy to `getSnapControllerMessenger`, this simplifies the tests a bit.

Reply:
I like what you did :+1: